src/components/DirectoryTree.tsx

```diff
@@ -317,6 +317,16 @@ export class DirectoryTree extends React.Component<DirectoryTreeProps, {
     this.lastClickedFile = file;
 
   }
+  timeout = 0;
+  layout = () => {
+    if (this.timeout) {
+      window.clearTimeout(this.timeout);
+    }
+    this.timeout = window.setTimeout(() => {
+      this.timeout = 0;
+      this.tree.layout();
+    }, 10);
```

Comment:
add a constant for magic number 10, e.g. `throttleTimeout`, before `this.timeout =` line.

also, `this.timeout` change to `this.layoutTreeThrottleTimer` provide more "documentation" to the code.